package.json

```diff
@@ -7,9 +7,11 @@
     "index.js"
   ],
   "scripts": {
-    "build": "style-dictionary build"
+    "build": "node build.js",
+    "transform": "token-transformer data/tokens.json data/transformed-tokens.json --expandTypography=true"
```

Comment:
Currently only expands the typography tokens, but can be added to expand box shadow tokens, etc. 

https://www.npmjs.com/package/token-transformer

Reply:
Excellent! I like how we can really use this to expand on the other tokens we'll be adding